use-cases/inferencing/benchmark/src/locust-tasks/logging.conf

```diff
@@ -0,0 +1,24 @@
+[formatters]
+keys=custom_json
+
+[formatter_custom_json]
+class=custom_json_formatter.CustomJSONFormatter
```

Comment:
You need to bring this file into your src directory

https://github.com/GoogleCloudPlatform/accelerated-platforms/blob/fix-locust-benchmarking/use-cases/model-fine-tuning-pipeline/data-processing/ray/src/custom_json_formatter.py